client/electron/process.ts

```diff
@@ -84,15 +92,20 @@ export class ChildProcessHelper {
 
         logExit(this.processName, code, signal);
         if (code === 0) {
-          resolve();
+          resolve(stdOutStr);
         } else if (code) {
           reject(new ProcessTerminatedExitCodeError(code, stdErrJSON));
         } else {
           reject(new ProcessTerminatedSignalError(signal ?? 'unknown'));
         }
       };
 
-      this.childProcess?.stdout?.on('data', data => this.stdOutListener?.(data));
+      this.childProcess?.stdout?.on('data', data => {
+        this.stdOutListener?.(data);
+        if (!ignoreStdOut) {
```

Comment:
This is a double-negative, which is a bit harder to read. Perhaps call it `returnStdOut`?

Reply:
Done.